spec.md

```diff
@@ -135,8 +144,8 @@ Subsections:
 - Describe caveats.
 - If using an external API generator (like go-doc, js-doc, or so on), point to an external `API.md` file. This can be the only item in the section, if present.
 
-#### Contribute
-**Status**: Required.  
+### Contribute
+**Status**: Required.
```

Comment:
Ah, I see, it's a "easier" way to write BR elements. http://www.wilfred.me.uk/blog/2012/07/30/why-markdown-is-not-my-favourite-language/

Can I replace them *with* BR elements, for the sake of the sanity of future editors?